packages/core/src/client/decorators/smartAccountClient.ts

```diff
@@ -54,19 +60,24 @@ export type BaseSmartAccountClientActions<
     args: SendUserOperationParameters<TAccount>
   ) => Promise<UserOperationStruct>;
   buildUserOperationFromTx: (
-    args: SendTransactionParameters<TChain, TAccount>
+    args: SendTransactionParameters<TChain, TAccount>,
+    overrides?: UserOperationOverrides
   ) => Promise<UserOperationStruct>;
   buildUserOperationFromTxs: (
     args: SendTransactionsParameters<TAccount>
   ) => Promise<BuildUserOperationFromTransactionsResult>;
```

Comment:
I think we should leave these two as is (I know it's confusing), but they were initially just meant to be used to convert a TX into a UO type that can be passed in to `sendUserOperation`. The thinking was that if a dev wanted to send a TX object as a UO, they weren't limited to just `sendTransaction`, but they could do `sendUserOperation()` with the result of `buildUserOperationFromTx(s)`